supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/io/package.scala

```diff
@@ -30,6 +30,7 @@ package object io {
 
   private [jes] def isFatalJesException(t: Throwable): Boolean = t match {
     case e: HttpResponseException if e.getStatusCode == 403 => true
+    case e: HttpResponseException if e.getStatusCode == 400 && e.getContent.contains("INVALID_ARGUMENT") => true
```

Comment:
@mcovarr -- This is the best I could do, but it feels ugly.  I was surprised that I couldn't find this as a string on its own